Formula/p/pgcopydb.rb

```diff
@@ -0,0 +1,30 @@
+class Pgcopydb < Formula
+  desc "Copy a Postgres database to a target Postgres server"
+  homepage "https://github.com/dimitri/pgcopydb"
+  url "https://github.com/dimitri/pgcopydb/archive/refs/tags/v0.17.tar.gz"
+  sha256 "7ed96f7bbc0a5250f3b73252c3a88d665df9c8101a89109cde774c3505882fdf"
+  license "PostgreSQL"
+  head "https://github.com/dimitri/pgcopydb.git", branch: "main"
+
+  on_linux do
+    depends_on "gcc"
+    depends_on "bdw-gc"
+    depends_on "libpq"
+  end
+  
+  depends_on "bdw-gc" => [:build]
+  depends_on "libpq" => [:build]
+  depends_on "sphinx-doc" => [:build]
+
+  def install
+    system "make", "clean"
+    system "make", "bin"
+    bin.install "src/bin/pgcopydb/pgcopydb"
+    system "make", "-C", "docs", "man"
+    man1.install Dir["docs/_build/man/*"]
+  end
+
+  test do
+    system bin/"pgcopydb", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Refactored test